spec/models/entry_filter_spec.rb

```diff
@@ -7,28 +7,70 @@
 
   it "#clients" do
     filter = EntryFilter.new
-
-    expect(filter.clients).to eq(Client.by_name)
```

Comment:
Since no `Clients` had been created in this test we were saying "expect [] to eq []".

In this test I'm asserting that `Client.by_name` is called, but another option would be to create some `Client`s and make sure they are returned by `EntryFilter#clients`. Happy to go with either approach 😄

Reply:
I think that checking that we're returning the correct `Client`s here makes more sense than checking if `Client` receives some method, as the latter is more likely to change in a future refactor.